### ISSUE TITLE
Boundary condition files wrong size time dimension

### ISSUE THREAD

**Issue by eastjames** [geoschem/integrated_methane_inversion#206]
Name: James East
Institution: Harvard ACMG

The `time` dimension on TROPOMI and blended boundary condition files on 2018-04-03 is the wrong size and HEMCO fails when reading these files as a result. This affects `v2024-01` and `v2024-03`.

Thanks to @laestrada for identifying the issue.

These files have 1 time step, but HEMCO expects 8. For example:
in
```bash
$ ncks -m -v time /n/holylfs05/LABS/jacob_lab/imi/ch4/blended-boundary-conditions/v2024-01/GEOSChem.BoundaryConditions.20180403_0000z.nc4
```
out
```
netcdf GEOSChem.BoundaryConditions.20180403_0000z {
  dimensions:
    time = UNLIMITED ; // (1 currently)
```

in
```bash
$ ncks -m -v time /n/holylfs05/LABS/jacob_lab/imi/ch4/blended-boundary-conditions/v2024-03/GEOSChem.BoundaryConditions.20180403_0000z.nc4
```
out
```
netcdf GEOSChem.BoundaryConditions.20180403_0000z {
  dimensions:
    time = UNLIMITED ; // (1 currently)
```

Should be:
in
```bash
ncks -m -v time /n/holylfs05/LABS/jacob_lab/imi/ch4/blended-boundary-conditions/v2024-03/GEOSChem.BoundaryConditions.20180404_0000z.nc4
```
out
```
netcdf GEOSChem.BoundaryConditions.20180404_0000z {
  dimensions:
    time = UNLIMITED ; // (8 currently)
```

in
```bash
ncks -m -v time /n/holylfs05/LABS/jacob_lab/imi/ch4/blended-boundary-conditions/v2024-01/GEOSChem.BoundaryConditions.20180404_0000z.nc4
```
out
```
netcdf GEOSChem.BoundaryConditions.20180404_0000z {
  dimensions:
    time = UNLIMITED ; // (8 currently)
```

**Comment by nicholasbalasus** [comment]
Thanks, James. That's really weird. If it isn't urgent, I can address this when I make new BCs in the next week or two. Can you start with 20180430 instead? The 20180401-20180429 files have no TROPOMI data for correction so they're not very useful.

**Comment by eastjames** [comment]
Thanks! I think it is not urgent so no problem to wait until you make new BCs. This came up during the 1-month spinup for the IMI test run, which is 20180501 - 20180508 and is easy to work around. We could consider different default testing dates.

**Comment by nicholasbalasus** [comment]
That's odd this hasn't come up before then!

**Comment by nicholasbalasus** [comment]
Closed by #227.
